docs/.vitepress/theme/custom.css

```diff
@@ -129,6 +159,225 @@ html.dark body {
+    max-height: 238px;
+    width: unset;
+  }
+}
+.VPHomeFeatures .item .VPLink {
+  --vp-c-bg-soft: #131B2D;
+  max-width: 557px;
+  max-height: 351px;
+  border-radius: 8px;
+}
+.VPHomeFeatures .item .VPLink .box {
+  display: grid;
+  grid-template-columns: 0.9fr 1.1fr;
+  grid-template-areas:
+    "title icon"
+    "details icon"
+    "link icon"
+    "link-text icon";
+  padding: 60px 38px 60px 38px;
+  }
+.VPHomeFeatures .item .VPLink .box .VPImage {
+  grid-area: icon;
+  justify-self: end;
+}
+.VPHomeFeatures .item .VPLink .box .title {
+  grid-area: title;
+  color: #F6D553;
+  font-size: 36px;
+  line-height: 52px;
+}
+.VPHomeFeatures .item .VPLink .box .details {
+--vp-c-text-2: #FFFBD5;
+  grid-area: details;
+  font-size: 22px;
+  line-height: 32px;
+}
+.VPHomeFeatures .item .VPLink .link-text {
+  grid-area: link;
+  color: #FFFBD5;
+}
+.VPHomeFeatures .item .VPLink .link-text-value {
+  --vp-c-text-2: #FFFBD5;
+  grid-area: link-text;
+  color: var(--vp-c-text-2);
+  font-size: 16px;
+  text-decoration: underline;
+}
+.VPButton {
+  --vp-button-brand-text: #FFFBD5;
+  color: var(--vp-button-brand-text);
+}
+.VPNavBarTitle .title {
+  border-bottom: none;
+}
+.VPNavBarTitle .title .VPImage {
+  height: 96px;
+}
+
+.VPNavBarTitle .title span {
+  display: none;
+}
+
+.VPContent .VPDoc .container .aside .aside-curtain {
+  display: none;
+}
+
+.VPNavBar .wrapper .container .content {
+  background-color: #131B2D;
+}
+
+.VPLink.VPNavBarMenuLink {
+  --vp_c-text-1: #FFFBD5;
+  color: var(--vp_c-text-1);
+  font-weight: 400;
+}
+.VPLink.VPNavBarMenuLink:hover {
+  --vp_c-text-1: #F6D553;
+  color: var(--vp_c-text-1);
+  
+}
+.VPLink.VPNavBarMenuLink:active {
+  --vp_c-text-1: #F6D553;
+  color: var(--vp_c-text-1);
+  text-decoration: underline;
+}
+.VPNavBarSearch #local-search .DocSearch {
+  background-color: transparent;
+}
+
+.DocSearch .DocSearch-Button-Keys {
+  display: none;
+}
+
+
+.DocSearch-Button-Container .DocSearch-Button-Placeholder {
+  display: none;
+}
+
+.DocSearch-Button .DocSearch-Search-Icon,
+.DocSearch-Button-Container .vp-icon {
+  --vp-c-text-2: #FFFBD5;
+  color: var(--vp-c-text-2);
+}
+
+.VPNavBarHamburger {
+  --vp-c-text-1: #FFFBD5;
+}
+
+.VPNavBarHamburger.active {
+  --vp-c-text-2: #F6D553;
+  color: var(--vp-c-text-1);
+}
+
+.DocSearch-Button:hover {
+  border-color: transparent;
+}
+
+.DocSearch-Button:hover .DocSearch-Search-Icon {
+  color:#F6D553;
+}
+
+.VPNavBarAppearance.appearance,.VPNavScreenAppearance.appearance {
+ display: none;
+}
+
+.VPSocialLinks.VPNavBarSocialLinks {
+  display: none;
+ }
+
+ @media (min-width: 768px) {
+  .VPNavBar .wrapper .container .content .content-body {
+    display: grid;
+    grid-template-areas: 
+      "navlinks search";
+    grid-template-columns: 1fr auto;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+  }
+
+  .VPNavBarSearch {
+    grid-area: search;
+    display: flex;
+    align-items: center;
+    padding-left: 0;
+  }
```

Comment:
Not sure which part is responsible for this, but when tabbing from the top on big screen, I first get focus on the Warp logo, then on the search icon which is at the end of the nav bar, and only then to the nav links.

Reply:
Ah, that might be because the .VPNavBarSearch `<div>` is placed right after the logo in the DOM 🤔  I'll look into it! 👍

Reply:
<img width="919" alt="image" src="https://github.com/user-attachments/assets/72e2fb18-0c02-48c1-8a95-0e57bfae7c1a">
this ::after varies in height for some reason, should prob always be the same height as the wave